### PR TITLE
 Fix 77 - Create Contact Page

### PR DESCRIPTION
**Pull request by ryanleesipes**
The first stab at this, created a contact page and updated footer link the proper way to point at the new page.

Let me know what you think, screenshots below:

screenshot from 2018-12-05 11-52-09

screenshot from 2018-12-05 11-52-25

### CODE REVIEW

**Comment by ryanleesipes** [comment]
This addresses #77

**Comment by ryanleesipes** [comment]
New "Get Support" sub-heading -> "Ask a Question".
screenshot from 2018-12-18 10-56-12

**Comment by ryanleesipes** [comment]
@mkmelin where do you mean? Not sure I understand.

**Comment by mkmelin** [comment]
For the whole file, looks like you use 4 (or 5?) spaces to indent elements. 

So instead of
<header id="main-feature">
     <h1>

That should be
<header id="main-feature">
  <h1>

**Comment by wsmwk** [comment]
The general direction SGTM.  

Would it make sense to specifically call out "Get Involved", as it is certainly another reason why someone would want to contact us?   (Yes, it's in the header menu, but it might be missed.)

**Comment by Sancus** [comment]
Yes, 2 space indents should be used in this file instead of 4.

I'll do a full review tomorrow but other than that nothing jumps out at me.

File: website/contact/index.html [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+            <div class="col full">
+                <p>{{ _('How to get support for Thunderbird.') }}</p>
+            </div>
+        </div>
+        <div class="row">
+            <div class="col full">
+                <section>
+                    <h3>{{ _('Mozilla Support Website') }}</h3>
+                    <p>{{ _('The best place to start looking for answers if you have questions about Thunderbird is the Mozilla Support website, <a href="%(support)s">specifically the Thunderbird page</a>.
+                      There you can search for articles, ask questions, and read tutorials.')|format(support='https://support.mozilla.org/products/thunderbird') }}</p>
```
Comment (Sancus) [inline website/contact/index.html]: This should use the url shortcut that already exists for the support site. `format(support=url('support'))`.

[the remark this answers is not in the document]
Comment (ryanleesipes) [reply website/contact/index.html]: Done. PR commit incoming.

File: website/contact/index.html [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+        <h2>{{ _('Get Support') }}</h2>
+        <div class="row">
+            <div class="col full">
+                <p>{{ _('How to get support for Thunderbird.') }}</p>
+            </div>
+        </div>
+        <div class="row">
+            <div class="col full">
+                <section>
+                    <h3>{{ _('Mozilla Support Website') }}</h3>
```
[the remark this answers is not in the document]
Comment (ryanleesipes) [reply website/contact/index.html]: Agreed.

File: website/contact/index.html [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+        <h2>{{ _('Partnerships') }}</h2>
+        <div class="row">
+            <div class="col full">
+                <p>{{ _('Partner with the Thunderbird Project.') }}</p>
+            </div>
+        </div>
+        <div class="row">
+            <div class="col full">
+                <section>
+                    <p>{{ _('Are you interested in partnering with the Thunderbird project? Do you provide services that could improve our the experience of using Thunderbird?
```
[the remark this answers is not in the document]
Comment (ryanleesipes) [reply website/contact/index.html]: Yup, typo or something.

File: website/contact/index.html [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+        <div class="row">
+            <div class="col full">
+                <section>
+                    <h3>{{ _('Mozilla Support Website') }}</h3>
+                    <p>{{ _('The best place to start looking for answers if you have questions about Thunderbird is the Mozilla Support website, <a href="%(support)s">specifically the Thunderbird page</a>.
+                      There you can search for articles, ask questions, and read tutorials.')|format(support='https://support.mozilla.org/products/thunderbird') }}</p>
+                </section>
+                <section>
+                    <h3>{{ _('Ask a Question') }}</h3>
+                    <p>{{ _('You can <a href="https://support.mozilla.org/questions/new/thunderbird">ask a question</a> on the support website. <b>Remember to be courteous</b>, as most of the people providing support on this site are volunteers and fellow Thunderbird users like you!') }}</p>
```
[the remark this answers is not in the document]
Comment (ryanleesipes) [reply website/contact/index.html]: As you pointed out, already there. Put it in instead.

File: website/contact/index.html [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+        <h2>{{ _('Press Inquiries') }}</h2>
+        <div class="row">
+            <div class="col full">
+                <p>{{ _('Information on how members of the press to reach out to us.') }}</p>
+            </div>
+        </div>
+        <div class="row">
+            <div class="col full">
+                <section>
+                    <p>{{ _('Are you a member of the press who would like to ask us a question about Thunderbird? You can reach our team at <a href="mailto:press@thunderbird.net">press@thunderbird.net</a>.') }}</p>
```
[the remark this answers is not in the document]
Comment (ryanleesipes) [reply website/contact/index.html]: Good point, I'll think on this a bit.

[the remark this answers is not in the document]
Comment (ryanleesipes) [reply website/contact/index.html]: Let's do the same as Mozilla.

**Pull request merged**